### ISSUE TITLE
"Proxy all traffic" should only be selectable if "Mange system proxy" is on.

### ISSUE THREAD

**Issue by p3rcya** [getlantern/lantern#5335]
Users are confused between "Proxy all traffic" and "Mange system proxy". To be honest, I was confused before as well. Putting "Proxy all traffic" as a sub-selection under "Mange system proxy" should clarify it

**Comment by stelectronic** [comment]
I agree. If "Manage system proxy" is off, "Proxy all traffic" is on or off have the same output. So why put the option " "Proxy all traffic" there? It can be canceled. I havn't figured out the difference that "Proxy all traffic" make. 

And I noticed a little bug. If "Mange system proxy" is on, and then you close the script PAC manually in system settingslike that:
![image](https://cloud.githubusercontent.com/assets/18217778/19178489/ad727a34-8c85-11e6-831e-2debb2799b44.png)

Lantern will not detect this change and the website http://127.0.0.1:XXXXX can't be accessed.

**Comment by p3rcya** [comment]
I think this UI can be integrated with the new UI for port info https://github.com/getlantern/lantern-internal/issues/251
